ui/page/wallets/wallet_page.go

```diff
@@ -368,6 +368,7 @@ func (pg *WalletPage) showAddWalletModal(l *load.Load) {
 				pg.loadWalletAndAccounts()
 				pg.Toast.Notify("Wallet created")
 				m.Dismiss()
+				pg.ChangeFragment(privacy.NewSetupPrivacyPage(pg.Load, wal))
```

Comment:
same way you have this in the goroutine, same should apply to the start page... but my recommendation will be that the moment a new wallet is created, navigate to the main page.
after which the automatic privacy setup process to kick in.